### PR TITLE
`Quiz exercises`: Fix issue with drag and drop quiz exercise creation

### PR DESCRIPTION
**Pull request by minrows**
#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

#### Client
- [x] I added multiple screenshots/screencasts of my UI changes.

### Motivation and Context
This PR fixes the issue that is there while creating drag and drop quiz exercises.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Course

1. Log in to Artemis
2. Navigate to Course Administration
3. Goto Exercises of one of the courses
4. Create Drag and Drop Model Quiz by clicking on the `Create Drag and Drop Model Quiz` button followed by `Create a new Apollon Diagram`
5. Drag and drop a few elements to the canvas and click on the `Interactive ` button from the side menu.
6. Click on a few elements until it's highlighted to green then click on the `Generate a Quiz Exercise` button (As illustrated in screenshot section)
7. Observe that there are no errors, and the quiz is generated successfully.

#### Code Review
- [ ] Review 1
- [ ] Review 2
#### Manual Tests
- [ ] Test 1
- [ ] Test 2

#### Before
screen-capture (13)

#### After
screen-capture (15)

### CODE REVIEW

File: src/main/webapp/app/entities/quiz/quiz-exercise.model.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ export class QuizExercise extends Exercise {
     public quizPointStatistic?: QuizPointStatistic;
     public quizQuestions?: QuizQuestion[];
     public status?: QuizStatus;
-    public quizMode?: QuizMode;
+    public quizMode?: QuizMode = QuizMode.INDIVIDUAL; // default value
```
Comment (ge65cer) [inline src/main/webapp/app/entities/quiz/quiz-exercise.model.ts:45]: If you provide a default value, does this still need to be an optional?

**Review by ge65cer — APPROVED** [review]
Tested on TS1, bug is fixed.

File: src/main/webapp/app/entities/quiz/quiz-exercise.model.ts (hunk 1 above)
Comment (minrows) [reply src/main/webapp/app/entities/quiz/quiz-exercise.model.ts:45]: Hi @ge65cer, thank you for your feedback.
-  I did try making it non optional but I was getting error as `EntityArraryResponseType of exercise.service` is not assignable to `EntityArraryResponseType of quiz-exercise.service`
-  I was aiming to fix the bug with least few changes as possible, hence I decided to make it optional as such (plus similar thing was already done [here](https://github.com/ls1intum/Artemis/blob/edee8b4bbefbf5f6a3373fe47475c516139a7e1e/src/main/webapp/app/entities/exercise.model.ts#L85) in the code.

**Review by Santia-go — APPROVED** [review]
Tested on TS1 
Working as expected.

**Review by stefanwaldhauser — APPROVED** [review]

**Pull request merged**